api/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+  "github.com/Salz-Team/salz/api/controllers"
+  "github.com/Salz-Team/salz/api/middlewares"
+  "github.com/Salz-Team/salz/api/config"
+  "github.com/gin-gonic/gin"
+)
+
+func main() {
+  // TODO Dependency injection?
+  // TODO config stuff
+  cfg := config.NewConfig()
+  ctrl := controllers.NewController(cfg)
```

Comment:
is this idiomatic? idfk

Reply:
Thinking back, I think I've seen this sort of dance from one of the popular cli framework libs, so this should be pretty acceptable.